articles/architecture-scenarios/b2c/phase-1-overview.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Consumer Identity and Access Management Phase 1 - Overview
```

Comment:
The content currently here was part of `Planning` in the original B2C Welcome; this content belongs under `Planning` otherwise there is no context for `Phase 1`. 

The content is also out of date; the content that should be used for planning phase 1 can be found here: https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.grf348t1jxv7

Reply:
The content here is also out of date; the content that should be used for planning phase 1 can be found here: https://docs.google.com/document/d/1wHEmRsPVciJJVuyssBH3RyO1mLYIGqLj34-vhVFEODU/edit#bookmark=id.grf348t1jxv7

Reply:
I'll update based on the updated google doc.